src/Module.Server/ModuleData/dtv/dtv_characters.xml

```diff
@@ -3929,7 +3929,7 @@
       <skill id="IronFlesh" value="5" />
       <skill id="PowerStrike" value="5" />
       <skill id="PowerDraw" value="0" />
-      <skill id="Athletics" value="232" />
```

Comment:
The shielders are at 120.
The line 3932 belongs to the Kuyak Hero which is a agi-2h-bot; 15/33 - they're supposed to be fast since so few of them spawn